transformation/standardJavascriptTransforms/src/dataStreamBackingIndexTransform.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Transforms data stream backing index operations for migration by replacing the
+ * backing index name with the data stream name and setting op_type to "create".
```

Comment:
Can you expand upon this a little bit more.  Assume that somebody finds this, doesn't know what datastreams are at all, and decides that this sounds like it's exactly what they need - even though it isn't!